components.yaml

```
@@ -5,54 +5,54 @@ GEOSadas:
 env:
   local: ./@env
   remote: ../ESMA_env.git
-  tag: v1.4.3
+  tag: v3.2.1
   develop: main
 
 cmake:
   local: ./@cmake
   remote: ../ESMA_cmake.git
-  tag: v1.0.11
+  tag: v3.4.2
   develop: develop
 
 ecbuild:
   local: ./@cmake/@ecbuild
   remote: ../ecbuild.git
-  tag: geos/v1.0.0
+  tag: geos/v1.0.6
 
 NCEP_Shared:
   local: ./src/Shared/@NCEP_Shared
   remote: ../NCEP_Shared.git
-  tag: v1.1.0
+  tag: rt-GEOSadas-5_29_0_prep3
```

@rtodling, you can change this to the v5.29.0 tag for NCEP_Shared:
```suggestion
  tag: v5.29.0
```
as the two tags are identical: https://github.com/GEOS-ESM/NCEP_Shared/compare/v5.29.0..rt-GEOSadas-5_29_0_prep3